.github/actions/build-vars/action.yml

```diff
@@ -49,30 +49,6 @@ runs:
       shell: bash
       run: echo "env_name=demo3" >> $GITHUB_OUTPUT
 
-    - name: Set Build Environment - trialfrontend01
```

Comment:
@JosiahSiegel Since we're going to be triggering trial frontend deployments from different branches (rather than `trialfrontend*` branches), these sections aren't really needed anymore.  Let me know if this is an issue!